src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const dayRate = 40;
+
+  const total = days * dayRate;
```

Comment:
total could be more descriptive. I could thing that it is total days for instant or total discount...
Check checklist p.1)